src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.figure.Circle;
+import core.basesyntax.figure.Figure;
+import core.basesyntax.figure.IsoscelesTrapezoid;
+import core.basesyntax.figure.Rectangle;
+import core.basesyntax.figure.RightTriangle;
+import core.basesyntax.figure.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int CASE_INDEX = 5;
+    public static final Color DEFAULT_COLOR = Color.WHITE;
+
+    private static final double DEFAULT_RADIUS = 10.0;
```

Comment:
none of these vars are used outside of that class
```suggestion
    private static final int CASE_INDEX = 5;
    private static final Color DEFAULT_COLOR = Color.WHITE;
    private static final double DEFAULT_RADIUS = 10.0;
```